joern-cli/frontends/csharpsrc2cpg/src/main/resources/builtin_types/Microsoft.AspNetCore.Mvc/builtin_types.json

```diff
@@ -0,0 +1,69 @@
+{
+  "Microsoft.Extensions.DependencyInjection": [
+    {
+      "name": "Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions",
+      "methods": [
+        {
+          "name": "AddMvc",
+          "returnType": "Microsoft.Extensions.DependencyInjection.IMvcBuilder",
+          "parameterTypes": [
+            [
+              "services",
+              "Microsoft.Extensions.DependencyInjection.IServiceCollection"
+            ]
+          ],
+          "isStatic": true
+        },
+        {
+          "name": "AddMvc",
+          "returnType": "Microsoft.Extensions.DependencyInjection.IMvcBuilder",
+          "parameterTypes": [
+            [
+              "services",
+              "Microsoft.Extensions.DependencyInjection.IServiceCollection"
+            ],
+            [
+              "setupAction",
+              "System.Action`1<Microsoft.AspNetCore.Mvc.MvcOptions>"
+            ]
+          ],
+          "isStatic": true
+        }
+      ],
+      "fields": []
+    },
+    {
+      "name": "Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions/FrameworkAssemblyPart",
```

Comment:
When I look for the types suffixed with `/` I don't seem to find them in the documentation, but I do find the parent. https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.mvcservicecollectionextensions?view=aspnetcore-7.0

Perhaps we can shorten these collections by removing those with `/` components?

Reply:
Added this filter in https://github.com/joernio/DotNetAstGen/pull/20. Will update this shortly. You can merge  https://github.com/joernio/DotNetAstGen/pull/20 🙏